R/fotmob_leagues.R

```diff
@@ -246,11 +246,10 @@ fotmob_get_league_matches <- function(country, league_name, league_id, cached =
 .fotmob_get_league_matches <- function(league_id, page_url) {
   resp <- .fotmob_get_league_resp(league_id, page_url)
   rounds <- resp$matches$data$matchesCombinedByRound
-  rounds %>%
-    purrr::map_dfr(
```

Comment:
there used to be an extra layer of nesting here that is no longer there, so we just need one `map_dfr`